src/react/GoogleMap.jsx

```diff
@@ -1,53 +1,48 @@
-import React, { useState, useCallback } from 'react'
+import React, { useState, useEffect } from 'react'
 import {
   APIProvider, Map, AdvancedMarker, Pin, InfoWindow
 } from '@vis.gl/react-google-maps'
-import UserLocation from './UserLocation'
```

Comment:
Please restore the geolocation functionality to display the user's current location on the map

You rewrote my function updatePosition to use `review` and `review.location` props to display markers and info.
The geolocation functionality is lost.

Reply:
I brought back your version of GoogleMap and merge conflicts are gone now